math/prime_factorization.cpp

```diff
@@ -40,7 +40,7 @@ void SieveOfEratosthenes(int N) {
 void prime_factorization(int num) {
     int number = num;
 
-    for (int i = 0; prime_numbers[i] <= num; i++) {
+    for (int i = 0; prime_numbers[i] <= number; i++) {
```

Comment:
This will actually increase the number of operations that are required. If you look inside the `while` loop inside this block on line 51, you would notice that number doesn't remain the same. Hence your change actually makes it calculate ALL the prime numbers until the initial number is reached.